azure-pipelines-integration-dartlab.yml

```diff
@@ -0,0 +1,72 @@
+    testExecutionJobStrategy:
+      maxParallel: 4
+      matrix:
+        debug_32:
+          _configuration: Debug
+          _oop64bit: false
+        debug_64:
+          _configuration: Debug
+          _oop64bit: true
+        release_32:
+          _configuration: Release
+          _oop64bit: false
+        release_64:
+          _configuration: Release
+          _oop64bit: true
+    testMachineTotalCount: 4
```

Comment:
This section may be Roslyn specific. It isn't needed if you aren't running your tests under a number of configurations.